robocop/checkers/lengths.py

```diff
@@ -15,57 +15,53 @@
         rule_id="0501",
         name="too-long-keyword",
-        msg="Keyword is too long (%d/%d)",
+        msg="Keyword is too long ({{ keyword_length }}/{{ allowed_length}})",
         severity=RuleSeverity.WARNING,
-        docs_args=(
-            "keyword length",
-            "allowed length",
-        ),
     ),
     "0502": Rule(
         RuleParam(name="min_calls", default=1, converter=int, desc="number of keyword calls required in a keyword"),
         rule_id="0502",
         name="too-few-calls-in-keyword",
-        msg="Keyword has too few keywords inside (%d/%d)",
+        msg="Keyword has too few keywords inside ({{ keyword_count }}/{{ min_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
     "0503": Rule(
         RuleParam(name="max_calls", default=10, converter=int, desc="number of keyword calls allowed in a keyword"),
         rule_id="0503",
         name="too-many-calls-in-keyword",
-        msg="Keyword has too many keywords inside (%d/%d)",
+        msg="Keyword has too many keywords inside ({{ keyword_count }}/{{ max_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
     "0504": Rule(
         RuleParam(name="max_len", default=20, converter=int, desc="number of lines allowed in a test case"),
         rule_id="0504",
         name="too-long-test-case",
-        msg="Test case is too long (%d/%d)",
+        msg="Test case is too long ({{ test_length }}/{{ allowed_length }})",
         severity=RuleSeverity.WARNING,
     ),
     "0505": Rule(
         RuleParam(name="max_calls", default=10, converter=int, desc="number of keyword calls allowed in a test case"),
         rule_id="0505",
         name="too-many-calls-in-test-case",
-        msg="Test case has too many keywords inside (%d/%d)",
+        msg="Test case has too many keywords inside ({{ keyword_count }}/{{ max_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
     "0506": Rule(
         RuleParam(name="max_lines", default=400, converter=int, desc="number of lines allowed in a file"),
         rule_id="0506",
         name="file-too-long",
-        msg="File has too many lines (%d/%d)",
+        msg="File has too many lines ({{ lines_count }}/{{max_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
     "0507": Rule(
         RuleParam(name="max_args", default=5, converter=int, desc="number of lines allowed in a file"),
         rule_id="0507",
         name="too-many-arguments",
-        msg="Keyword has too many arguments (%d/%d)",
+        msg="Keyword has too many arguments ({{ arguments_count }}/{{ max_allowed_count }})",
         severity=RuleSeverity.WARNING,
     ),
     "0508": Rule(
-        RuleParam(name="line_length", default=120, converter=int, desc="number of lines allowed in a file"),
+        RuleParam(name="line_length", default=120, converter=int, desc="number of lines allowed in a file"),  # FIXME
```

Comment:
```suggestion
        RuleParam(name="line_length", default=120, converter=int, desc="number of lines allowed in a file"),
```

Reply:
FIXME is because the message content is wrong - it's not "number of lines allowed in a file" but "number of characters allowed in line" (but I didn't have any inspiration for properly naming it so I tagged it with FIXME)

Reply:
"number of characters allowed in line" looks fine 😃

Reply:
See, it took me 10 mins yesterday and I couldn't figure it what I did here in the mid of conversation ;)